res/drawable-night/message_request_button_background.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this is forcing dark mode for the buttons if the system is in night mode, but app could be in "light" theme which should force light theme ignoring system theme

Reply:
I'll figure it out if not, but do you know how to fix this?

Reply:
adding an attribute in `attrs.xml` then in `themes.xml` add an item to dark and light themes each one using the appropriate drawable, the property could be called something like "message_request_button_bg" and then name the drawables `message_request_button_bg_light.xml` and `message_request_button_bg_dark.xml`

Reply:
Nevermind, I couldn't wait and figured it out myself :joy: (did it like Signal)

Reply:
This added 6ms to startup time, but I think that's OK, as we will also be able to use more `-night` views in the future

Reply:
Wait, you had already answered, but I didn't see it. Anyway, I it's solved now, but in another way :shrug: